packages/testing/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@firebase/testing",
+  "version": "0.1.0",
+  "description": "",
+  "author": "Firebase <firebase-support@google.com> (https://firebase.google.com/)",
+  "main": "dist/cjs/index.js",
+  "browser": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "scripts": {
+    "dev": "gulp dev",
+    "test": "run-p test:node",
```

Comment:
If you don't have any browser tests you can omit the `npm-run-all` dependency, and replace this `test` script with the one in `test:node`

Reply:
done